plugins/single_plugins/fast-switcher.cpp

```diff
@@ -0,0 +1,194 @@
+#include <output.hpp>
+#include <debug.hpp>
```

Comment:
it is highly probable I left this aside, but if you're not debugging, it's best to remove this